config/hardware/accelerometers/adxl345_btt_lis2dw.cfg

```
@@ -0,0 +1,26 @@
+# This ADXL file is dedicated to be used with ADXL boards
+# connected over USB to the pi as dedicated and standalone ADXL-MCU boards
+
+# https://github.com/bigtreetech/LIS2DW
+
+# You need to override the following to be able to set the proper serial in your overrides.cfg file
+[mcu adxl]
+serial: /dev/serial/by-id/xxx
+
+[adxl345]
```

This need to be changed to use the proper Klipper section for this kind of chip

```suggestion
[lis2dw]
```